homeassistant/bootstrap.py

```diff
@@ -510,10 +511,12 @@ async def _async_set_up_integrations(
 
     stage_2_domains = domains_to_setup - logging_domains - debuggers - stage_1_domains
 
-    # Kick off loading the registries. They don't need to be awaited.
-    asyncio.create_task(hass.helpers.device_registry.async_get_registry())
-    asyncio.create_task(hass.helpers.entity_registry.async_get_registry())
-    asyncio.create_task(hass.helpers.area_registry.async_get_registry())
+    # Load the registries
+    asyncio.gather(
```

Comment:
We probably need to await this at some point if we want it to be ready before leaving the setup phase.

Reply:
Yes, absolutely, fixed in cd5f7e5